src/aura/HH_Container/HH_ContainerHelper.js

```diff
@@ -427,4 +446,145 @@
+        var hhId = component.get('v.hhId');
+        var hhTypePrefix = component.get('v.hhTypePrefix');        
+        var listCon = component.get("v.listCon");
+        
+        if (hhTypePrefix == '001')
+            conAdd.AccountId = hhId;
+        else
+            conAdd.npo02__Household__c = hhId;
+        // put them at the end of our naming list
+        conAdd.npo02__Household_Naming_Order__c = listCon.length;
+        listCon.push(conAdd);
+		component.set('v.listCon', listCon);    
+
+        // update the contact's addresses to the hh default (if they have no override)
+        var addrDefault = component.find('addrMgr').get('v.addrDefault');
+        if (addrDefault != null)
+            this.updateDefaultAddress(component, addrDefault);
+        
+        // UNDONE: need to add the contact's address to our address mgr (if not override)
+
+        // force our names to update since we have new contacts!
+        this.updateHHNames(component);
+        
+        // add the Contact's address to our list 
+        var action = component.get("c.addContactAddresses");
+        var listAddr = component.get('v.listAddr');
+        // because our address objects aren't real, we need to 
+        // tell the system what type of sobject they are
+        for (var i in listAddr)
+            listAddr[i].sobjectType = 'Address__c';
+        var listCon = [conAdd];
+        action.setParams({ listCon : listCon, listAddrExisting : listAddr });
+        var self = this;
+        action.setCallback(this, function(response) {
+            var state = response.getState();
+            if (component.isValid() && state === "SUCCESS") {
+                var listAddr = response.getReturnValue();
+                component.set("v.listAddr", listAddr);                
+            }
+            else if (component.isValid() && state === "ERROR") {
+                self.reportError(component, response);
+            }            
+        });
+        $A.enqueueAction(action);        
+        
+    },
+
+    /*******************************************************************************************************
+    * @description merge the specified household and its contacts into the existing household
+    */
+	mergeHousehold : function(component, hhMerge) {
+        component.set("v.showSpinner", true);
+        var hhId = component.get('v.hhId');
+        var hhTypePrefix = component.get('v.hhTypePrefix');        
+
+        // query for the Household Contacts 
+		var action = component.get("c.getContacts");
+        action.setParams({ hhId : hhMerge.Id });
+		var self = this;
+		action.setCallback(this, function(response) {
+			var state = response.getState();            
+			if (component.isValid() && state === "SUCCESS") {
+                var listConMerge = response.getReturnValue();
+				var listCon = component.get("v.listCon");
+                
+				// move all contacts into our household
+				var cExisting = listCon.length;
```

Comment:
no.  it is passed in when the component is instantiated, and everywhere it is used without additional checks to see if it is null.  even if they delete all members of the household or there are no members originally, it will be an array with no members.